src/app/system/manage-dashboards/manage-dashboards.component.scss

```
@@ -0,0 +1,967 @@
+  .action-bar-left {
+    display: flex;
+    align-items: center;
+    gap: 16px;
+    flex-wrap: wrap;
+  }
+
+  .compact-office-select {
+    width: 288px;
+    margin-bottom: 0;
+
+    ::ng-deep {
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)